### PR TITLE
fix(ecommerce): OS-6979 update error translation for 30071 #patch

### PR DESCRIPTION
**Pull request by kraiztmatte**
update translation to support both bundle and option items since both of item using the same code error,

### CODE REVIEW

File: src/lib/service-error-translator/error-translation-map/ecommerce-admin-error-translation-map.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { Trans } from "react-i18next";
 export const EcommerceAdminErrorTranslationMap: { [key: string]: React.ReactNode } = {
   30071: (
     <Trans i18nKey="adminServiceError.30071">
-      This item cannot be deselected while the associated bundle is still selected
+      This item cannot be deselected while the associated bundle/option box is still selected
```
Comment (rayhan-ab) [inline src/lib/service-error-translator/error-translation-map/ecommerce-admin-error-translation-map.tsx:13]: update version in package.json and in the translation .json file too?

**Review by aylwin-AB — COMMENTED** [review]
is it intended to not add translation list on `en-US.json` and `zh-CN.json`?

File: src/lib/service-error-translator/error-translation-map/ecommerce-admin-error-translation-map.tsx (hunk 1 above)
Comment (kraiztmatte) [reply src/lib/service-error-translator/error-translation-map/ecommerce-admin-error-translation-map.tsx:13]: updated, thank you.

Comment (rayhan-ab) [reply src/lib/service-error-translator/error-translation-map/ecommerce-admin-error-translation-map.tsx:13]: version in package.json?

Comment (kraiztmatte) [reply src/lib/service-error-translator/error-translation-map/ecommerce-admin-error-translation-map.tsx:13]: ah yea sorrry updated, please review again

**Comment by kraiztmatte** [comment]
> is it intended to not add translation list on `en-US.json` and `zh-CN.json`?

i forgot to add it, now its updated :+1:

**Review by castrix — APPROVED** [review]

**Review by rayhan-ab — APPROVED** [review]

**Review by aylwin-AB — APPROVED** [review]

**Pull request merged**